NTupleAnalyzerHH_nanoAOD.py

```diff
@@ -1405,8 +1624,8 @@ def MediumIDMuons(T,_met,variation,isdata):
 		if (Pass):
 			NewMu = TLorentzVector()
 			OldMu = TLorentzVector()
-			NewMu.SetPtEtaPhiM(_MuonPt[n],T.Muon_eta[n],T.Muon_phi[n],0)
-			OldMu.SetPtEtaPhiM(T.Muon_pt[n],T.Muon_eta[n],T.Muon_phi[n],0)
+			NewMu.SetPtEtaPhiM(_MuonPt[n],T.Muon_eta[n],T.Muon_phi[n],T.Muon_mass[n])
```

Comment:
This looks like a problem in the indentation compared to the old version, please check.